packages/normalize/postcss.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  plugins: [
+    require('postcss-import'),
+    require('postcss-preset-env')({
+      browsers: 'last 2 versions, IE <= 10',
```

Comment:
does it matter that the browerlists in this pr are diff?

Reply:
yes....i need to remove those. good eye